table/block_based_table_reader.cc

```diff
@@ -1143,10 +1144,12 @@ FilterBlockReader* BlockBasedTable::ReadFilter(
   }
   BlockContents block;
 
-  BlockFetcher block_fetcher(
-      rep->file.get(), prefetch_buffer, rep->footer, ReadOptions(),
-      filter_handle, &block, rep->ioptions, false /* decompress */,
-      Slice() /*compression dict*/, rep->persistent_cache_options);
+  Slice dummy_comp_dict = Slice();
```

Comment:
Don't need explicit initialization just to be consistent with other places where you introduced dummy_comp_dict.